tests/test_single_topology.py

```diff
@@ -175,9 +170,29 @@ def testing_find_dummy_groups_and_multiple_anchors():
     core_b = [2, 1, 4, 3]
 
     with pytest.warns(MultipleAnchorWarning):
-        dgs, jks = single_topology.find_dummy_groups_and_anchors(mol_a, mol_b, core_a, core_b)
-        assert dgs == [{0}]
-        assert jks == [(1, 2)]
+        dgs = single_topology.find_dummy_groups_and_anchors(mol_a, mol_b, core_a, core_b)
+        assert dgs == {1: (2, {0})}
+
+
+def test_ethane_cyclobutane():
```

Comment:
nit: This is cyclobutadiene, cyclobutane is C1CCC1

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/1019/commits/f2cdeaaf1f22b948c4e97d9bfbbc829e98499818